components/Modal/src/Modal.tsx

```diff
@@ -9,7 +9,7 @@ export interface ModalProps {
   open?: boolean;
   title?: string;
   closeLabel?: string;
-  children?: React.ReactNode;
+  children?: (toggle: (toggle?: boolean) => void) => React.ReactNode;
```

Comment:
Je zou ook nog dit kunnen doen
```suggestion
  children?: (toggle: (toggle?: boolean) => void) => React.ReactNode | React.ReactNode;
```

Reply:
Ah ik denk dat het zo geschreven moet worden:
```suggestion
 [ children?: (toggle: (toggle?: boolean) => void) => React.ReactNode;](children: ((toggle: (toggle: boolean) => void) => React.ReactNode) | React.ReactNode;)
```